packages/file-search/src/browser/quick-file-open.ts

```diff
@@ -30,13 +30,22 @@ import { NavigationLocationService } from '@theia/editor/lib/browser/navigation/
 import * as fuzzy from '@theia/core/shared/fuzzy';
 import { MessageService } from '@theia/core/lib/common/message-service';
 import { FileSystemPreferences } from '@theia/filesystem/lib/browser';
+import { EditorOpenerOptions, Position, Range } from '@theia/editor/lib/browser';
 
 export const quickFileOpen: Command = {
     id: 'file-search.openFile',
     category: 'File',
     label: 'Open File...'
 };
 
+export interface FilterAndRange {
+    filter: string;
+    range: Range;
+}
+
+// Supports patterns of <path><#|:><line><#|:|,><col?>
+const LINE_COLON_PATTERN = /\s?[#:\(](?:line )?(\d*)(?:[#:,](\d*))?\)?\s*$/;
```

Comment:
I understand that the `regexp` come from vscode, but here don't we mean `column` and not `colon`?

```suggestion
const LINE_COLUMN_PATTERN = /\s?[#:\(](?:line )?(\d*)(?:[#:,](\d*))?\)?\s*$/;
```

I believe that `col` in `<path><#|:><line><#|:|,><col?>` is the column.

Reply:
Done, I agree, it makes more sense as LINE_COLUMN_PATTERN, thanks !

Reply:
nitpicking but I think `LINE_COLON_PATTERN` actually meant patterns containing a colon like `path:line` but not sure.

Reply:
I think both names work fine, so I don't really have a strong opinion either way, 
@marechal-p and @vince-fugnitto any strong opinions either way?

Reply:
we can go with what @marechal-p proposed, it’s the same name as in vscode as well :)

Reply:
Done, the latest commit reverted the name back to: `LINE_COLON_PATTERN`